### DISCUSSION TITLE
How to allow cells to have multi-line text?

### DISCUSSION THREAD

**Discussion by rau** [iddan/react-spreadsheet#382] [Q&A]
Hey, currently trying to make my cells work with multi-line text (i.e. respect whitespace pre-wrap).

I can't figure out how to do that. Here's the code I'm trying, but it's not working.

Is there a prop I can enable on the spreadsheet component that allows my cells to have multi-line text?

```
<Spreadsheet
  data={data}
  onChange={handleDataChange}
  onBlur={handleBlur}
  columnLabels={columnLabels}
  className='!whitespace-pre'
  hideRowIndicators={true}
/>
```

**Comment by wenerme** [comment]
juse include `\n` if you dont change the viewer
